Utils/Dataflow/060_upload2virtuoso/uploadTTL.sh

```diff
@@ -209,8 +211,12 @@ do
       MODE="${2,,}"
       shift
       ;;
-    -d|--delimiter)
-      DELIMITER=`echo -e $2`
+    -b|--batch
+      BATCHMODE="${2,,}"
+      shift
+      ;;
+    -B|--eob)
+      EOBatch=`echo -ne $2`
```

Comment:
I believe we have quite a long discussion about this mistake I\`ve done when wrote this script :)
What happens when `echo -ne $2` is used?
1. If `$2` was `"\0"` or `"\n"`, the value of `EOBatch` will be the empty string, as shell trims all empty/space symbols from command substitution result. See in the example with `\n`:
```
[mgolosova@bamboo dkb]$ echo -e "abcde\n\n\n"                                                                                                                                                                
abcde



[mgolosova@bamboo dkb]$ echo `echo -ne "abcde\n\n\n"`
abcde
[mgolosova@bamboo dkb]$ 
```

2. Even if `$2` was not a code of "empty/space" symbol: here you get the interpreted value, while the default value is not interpreted: `EOBatch='\x17'`. What do you pass to `upload_stream`? Interpreted value or not? If not -- what will be used as a delimiter in `read -r -d "$delimiter"`?
See:
```
[mgolosova@bamboo dkb]$ EOBatch='\x17'                                                                                                                                                                             
[mgolosova@bamboo dkb]$ echo -ne "message_1\nmessage_2\n${EOBatch}message_3\nmessage_4\n${EOBatch}" \
>   | hexdump -C
00000000  6d 65 73 73 61 67 65 5f  31 0a 6d 65 73 73 61 67  |message_1.messag|
00000010  65 5f 32 0a 17 6d 65 73  73 61 67 65 5f 33 0a 6d  |e_2..message_3.m|
00000020  65 73 73 61 67 65 5f 34  0a 17                    |essage_4..|
0000002a
[mgolosova@bamboo dkb]$ echo -ne "message_1\nmessage_2\n${EOBatch}message_3\nmessage_4\n${EOBatch}" \
>   | while read -r -d "$EOBatch" line; do echo $line; done
[mgolosova@bamboo dkb]$ echo -n "message_1\nmessage_2\n${EOBatch}message_3\nmessage_4\n${EOBatch}" \
>   | while read -r -d "$EOBatch" line; do echo $line; done                                                         
message_1
nmessage_2
n
x17message_3
nmessage_4
n
[mgolosova@bamboo dkb]$ echo -ne "message_1\nmessage_2\n${EOBatch}message_3\nmessage_4\n${EOBatch}" \
>   | while read -r -d "`echo -ne $EOBatch`" line; do echo $line; done                                             
message_1 message_2
message_3 message_4
[mgolosova@bamboo dkb]$ 
```

Resume:
* variable should always contain interpreted or not interpreted value, whenever it was (re)assigned;
* to `read` should be passed interpreted value;
* we have to do something with "empty/space" symbols: `read` uses `\0` as delimiter if empty string is passed, but I have no idea right now how to pass it `\n` :)